lib/TerraformDevKit/terraform_template_renderer.rb

```diff
@@ -0,0 +1,41 @@
+require 'fileutils'
+require 'TerraformDevKit/terraform_template_config_file'
+
+module TerraformDevKit
+  class TerraformTemplateRenderer
+    def initialize(env, project, extra_vars_proc)
+      @env = env
+      @project = project
+      @extra_vars_proc = extra_vars_proc
+    end
+
+    def render_files
+      template_dirs = Configuration.get('template-dirs')

```

Comment:
Does this throw if `template-dirs` isn't defined in the config?

Reply:
Nope. It just returns nil.